samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -7,61 +7,114 @@ const {
     TeamsInfo,
     TeamsActivityHandler,
     CardFactory,
-    ActionTypes
-} = require('botbuilder');
-const TextEncoder = require('util').TextEncoder;
+    ActionTypes,
+} = require("botbuilder");
+const TextEncoder = require("util").TextEncoder;
 
 class TeamsConversationBot extends TeamsActivityHandler {
     constructor() {
         super();
+
         this.onMessage(async (context, next) => {
             TurnContext.removeRecipientMention(context.activity);
             const text = context.activity.text.trim().toLocaleLowerCase();
-            if (text.includes('mention')) {
+            if (text.includes("mention")) {
                 await this.mentionActivityAsync(context);
-            } else if (text.includes('update')) {
+            } else if (text.includes("update")) {
                 await this.cardActivityAsync(context, true);
-            } else if (text.includes('delete')) {
+            } else if (text.includes("delete")) {
                 await this.deleteCardActivityAsync(context);
-            } else if (text.includes('message')) {
+            } else if (text.includes("message")) {
                 await this.messageAllMembersAsync(context);
-            } else if (text.includes('who')) {
+            } else if (text.includes("who")) {
                 await this.getSingleMember(context);
             } else {
                 await this.cardActivityAsync(context, false);
             }
         });
 
         this.onMembersAddedActivity(async (context, next) => {
+            //  console.log(context);
             context.activity.membersAdded.forEach(async (teamMember) => {
                 if (teamMember.id !== context.activity.recipient.id) {
-                    await context.sendActivity(`Welcome to the team ${ teamMember.givenName } ${ teamMember.surname }`);
+                    await context.sendActivity(
+                        `Welcome to the team ${teamMember.givenName} ${teamMember.surname}`
+                    );
                 }
             });
             await next();
         });
+
+        //Subscribe to different conversation events
+
+        this.onTeamsChannelCreatedEvent(async (context, next) => {
+            await this.onTeamsChannelCreated(context);
+            await next();
+        });
+
+        this.onTeamsChannelRenamedEvent(async (context, next) => {
+            await this.onTeamsChannelRenamed(context);
+            await next();
+        });
+        this.onTeamsChannelDeletedEvent(async (context, next) => {
+            await this.onTeamsChannelDeleted(context);
+            await next();
+        });
+        this.onTeamsChannelRestoredEvent(async (context, next) => {
+            await this.onTeamsChannelRestored(context);
+            await next();
+        });
+
+        this.onTeamsTeamRenamedEvent(async (context, next) => {
+            await this.onTeamsTeamRenamed(context);
+            await next();
+        });
```

Comment:
I don't *believe* these are necessary since [`dispatchConversationUpdateActivity()`](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder/src/teamsActivityHandler.ts#L447) should forward those events to the overrides you have below.

Reply:
Marking this as Resolved as wrong file went in this PR